Sources/JWTKeychain/Validators/Validators.swift

```diff
@@ -0,0 +1,37 @@
+import Fluent
+import Validation
+import Vapor
+
+public final class ValidName: Validator {
+    public init() {}
+
+    public func validate(_ input: String) throws {
+        // TODO: decide on good standards for name validation. 
+        guard input.characters.count > 0, input.characters.count < 256 else {
+            throw error("Name should be between 1 and 255 characters in length")
+        }
+    }
+}
+
+// TODO: replace with Sugar's version when it's available
+public final class StrongPassword: Validator {
```

Comment:
StrongPassword + Unique + Exists are in Sugar now and some of them might also be vapor/validation vapor/fluent-validation (I haven't checked for an overlap yet)